yt_dlp/extractor/slideslive.py

```diff
@@ -139,6 +277,51 @@ def _real_extract(self, url):
         assert service_name in ('url', 'yoda', 'vimeo', 'youtube')
         service_id = player_info['service_id']
 
+        slides, slides_xml, slide_id = None, None, 1
+        slide_url_template = self._JPG_SLIDE_TMPL
+        chapters, thumbnails = [], []
+        if url_or_none(player_info.get('thumbnail')):
+            thumbnails.append({'url': player_info['thumbnail']})
+
+        if player_info.get('slides_json_url'):
+            slides = traverse_obj(self._download_json(
+                player_info['slides_json_url'], video_id, fatal=False,
+                note='Downloading slides JSON', errnote=False), 'slides', expected_type=list)
+        if slides:
+            if re.match(r'.+(/v[45]/).+', player_info['slides_json_url']):
+                slide_url_template = self._PNG_SLIDE_TMPL
+            for slide in slides:
```

Comment:
My suggestion was assuming that `slide_id` is not needed. If it is, enumerate, not manually increment
```suggestion
            for slide_id, slide in enumerate(slides):
```

Reply:
I am finding that a `slide_id` value is needed because there are some "slides"/chapters without any actual slides, so the numbering of chapters vs. thumbs will get out of sync if the extractor does not explicitly set the id. I will enumerate though